File: Java/프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        char [] a = s.toCharArray();
        
        if(!(a.length==4) && !(a.length==6) ){
            return false;
        }
        for(int i =0; i<a.length;i++){
            if(a[i]>57 || a[i]<48){
                return false;
            }
        }
        return true;
    }
}

File: Kotlin/프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.kt
class Solution {
    fun solution(s: String): Boolean {
        var answer = true
        try{
            if(s.length != 4 && s.length != 6) answer = false
            s.toInt()
        }catch (e :NumberFormatException) {
            answer = false
        }
        return answer
    }
}